util/log/clog.go

```diff
@@ -844,7 +844,7 @@ func (l *loggingT) getTermColorProfile() *colorProfile {
 			case "ansi", "xterm-color":
 				l.colorProfile = colorProfile8
 				color = true
-			case "xterm-256color":
+			case "xterm-256color", "screen-256color":
```

Comment:
The right way would be to look in the terminfo database, but that means an extra dependency (unless we're picking one up automatically due to our CLI dependencies). Looking for a suffix of "-256color" sounds reasonable to me.